guides/common/modules/proc_creating-variables-for-convert2rhel-role.adoc

```diff
@@ -3,18 +3,18 @@
 
 Assigning the Ansible role to a host also installs variables.
 You must create the following additional variables:
-[cols="20%,10%,15%",options="header"]
+[cols="20%,10%,20%,15%",options="header"]
 |====
-| Name | Type | Default value and description
-| {project-context}_rhel_wait_for_syncs | boolean | false
-| {project-context}_manifest_path | string | path to manifest
-| {project-context}_organization | string | name of organization
-| {project-context}_password | string | your admin password
-| {project-context}_server_url | string | URL of {project-context} instance
-| {project-context}_username | string | admin
-| {project-context}_validate_certs |boolean | Set to `true` if you wish to enable certificate checks in Ansible (default: true)
-| {project-context}_content_rhel_enable_rhel7 | boolean | Set to `true` if converting to {RHEL} 7
-| {project-context}_content_rhel_enable_rhel8 | boolean | Set to `true` if converting to {RHEL} 8
+| Name | Type | Default value and description | Example
+| {project-context}_rhel_wait_for_syncs | boolean | false | false
+| {project-context}_manifest_path | string | path to manifest | /root/manifest_Sat_20220623T124844Z.zip
+| {project-context}_organization | string | name of organization | Default Organization
+| {project-context}_password | string | your admin password | _My_Admin_Password_
+| {project-context}_server_url | string | URL of {project-context} instance | {foreman-example-com}
```

Comment:
```suggestion
| {project-context}_server_url | string | {ProjectServer} URL | {foreman-example-com}
```

Looking at the output, this would IMHO be an improvement.